internal/commands/waypoint/templates/list.go

```diff
@@ -76,5 +76,11 @@ func listTemplates(opts *TemplateOpts) error {
 		templates = append(templates, resp.GetPayload().ApplicationTemplates...)
 	}
 
-	return opts.Output.Show(templates, format.Pretty)
+	templateFields := []format.Field{
```

Comment:
I'm good with reducing the # of fields displayed to just these 3! Makes sense that we give the user the name of the template to look up with `read` if they want more info.